superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -105,19 +129,22 @@ const RightMenu = ({
           label: t('Upload CSV to database'),
           name: 'Upload a CSV',
           url: '/csvtodatabaseview/form',
-          perm: CSV_EXTENSIONS,
+          perm: CSV_EXTENSIONS && showUploads,
+          upload: true,
```

Comment:
would these need to be variable for the upload param?

Reply:
I don't understand, why would they be?